Gyanthakur/Dev_Profiles
# Updated project card layout

## Description

Updated the div layout and kept a single div for project url, project_name and deply link

Fixes: #317 

Use [x] to represent a checked (ticked) box.✅
Use [ ] to represent an unchecked box.❌

## Type of Change


## How to Test

Provide a brief description of how to test your changes. Include steps, commands, or screenshots if necessary.

1. Got to the project page and you'll see the change inside cards with properly aligned url, project name, and deploy link

## Checklist


## Additional Notes

Adding screenshot of a row with cards with highlighted changes 
![image](https://github.com/user-attachments/assets/c4dd23c2-57ad-4021-b77c-b96603d14080)
